### PR TITLE
Fix bugs with Simpson's integration and position calculation with Simpson's integration

### PR DESCRIPTION
**Pull request by PhilipAxelrod**
Fix bug where position calculations with Simpson's integration used second derivative of position instead of first. In addition, fix bug where dt was twice as much as it should be. By coincidence, the unit test used incorrect values that showed this integration method as functioning properly.

### CODE REVIEW

**Review by nikwalia — APPROVED** [review]
LGTM?

File: commons/src/main/scala/com/lynbrookrobotics/potassium/commons/cartesianPosition/XYPosition.scala [hunk 1]
```diff
@@ -8,64 +8,68 @@ import squants.space._
...
-      Feet(0)
-    )
+      Feet(0))
 
     // approximate that robot traveled in a straight line at the average
     // angle over the course of 1 tick
     deltaDistance.zip(averageAngle).scanLeft(origin){
       case (acc, (distance, avrgAngle), _) =>
-        acc + new Point(
-          distance * Math.cos(avrgAngle.toRadians),
```
Comment (shadaj) [inline commons/src/main/scala/com/lynbrookrobotics/potassium/commons/cartesianPosition/XYPosition.scala:38]: If we're dropping use of `java.util.Math`, we should remove the import as well.

**Review by shadaj — CHANGES_REQUESTED** [review]
Generally looks good, just one cleanup we can include in this PR

File: commons/src/main/scala/com/lynbrookrobotics/potassium/commons/cartesianPosition/XYPosition.scala (hunk 1 above)
Comment (PhilipAxelrod) [reply commons/src/main/scala/com/lynbrookrobotics/potassium/commons/cartesianPosition/XYPosition.scala:38]: Done

**Review by shadaj — APPROVED** [review]
LGTM! Thanks for the fix @Paxelord.

**Pull request merged**